### PR TITLE
Issue/6439 (WIP) - update Grunt configuration

### PR DESCRIPTION
**Pull request by SDavisMedia**
See #6439

### CODE REVIEW

File: Gruntfile.js [hunk 1]
```diff
@@ -81,103 +121,60 @@ grunt.initConfig({
...
-				cmd: 'tx push -s'
-			},
+		// Clean up build directory
+		clean: {
+			main: ['build/<%= pkg.name %>']
 		},
 
-		dirs: {
-			lang: 'languages',
+		// Copy the theme into the build directory
```
Comment (pippinsplugins) [inline Gruntfile.js]: `theme` should be `plugin`

Comment (SDavisMedia) [reply Gruntfile.js]: oooops... fixed

**Comment by spencerfinnell** [comment]
The `copy` task might be simpler if you used a whitelist instead of a blacklist. Seems like theres just a few top level files and then a few directories vs excluding all the dotfiles.

https://plugins.trac.wordpress.org/browser/easy-digital-downloads/trunk

8 includes vs many excludes.

**Comment by SDavisMedia** [comment]
@spencerfinnell Good call. I agree. I ultimately decided to make the same type of change (conceptually) for the `cssmin` and `uglify` tasks. I'll revisit `copy` later on tonight.

**Pull request merged**